src/Chapter4.hs

```diff
@@ -610,7 +630,17 @@ Implement the 'Monad' instance for our lists.
 🕯 HINT: You probably will need to implement a helper function (or
   maybe a few) to flatten lists of lists to a single list.
 -}
+instance Monad List where
+  (>>=) :: List a -> (a -> List b) -> List b
+  (>>=) Empty _ = Empty
+  (>>=) (Cons x xs) f = listConcat (f x) (xs >>= f)
```

Comment:
Nice!
You can use `fmap` here and have just one case 🙂

Reply:
but in this case i need to implement flatten function, right?

Reply:
Oh, yeah, I thought that it is already here, nevermind 😄